pkg/operator/controller/ingress/deployment.go

```diff
@@ -1645,7 +1652,7 @@ func copyProbe(a, b *corev1.Probe) {
 			Port: a.ProbeHandler.HTTPGet.Port,
 			Host: a.ProbeHandler.HTTPGet.Host,
```

Comment:
`Port` is required, and the default value for `Host` is the empty string, so I believe we can copy them unconditionally.

Reply:
In that case, if you wouldn't mind, can you add a comment like `Values that are always updated` and group them, and another comment `Values that are only updated when requested` and group them all under a
`if copyDefaultValues {`.  I feel like this may help/force future developers to make a decision about what to do with values in `copyProbe`.

Reply:
Done in https://github.com/openshift/cluster-ingress-operator/pull/1052/commits/3faf391d2d43c63526b256e5f83b163673d8ef9a.